Sources/SCTE35/ModelObjects/SpliceDescriptors/SegmentationInfo.swift

```diff
@@ -343,75 +349,214 @@ public struct SegmentationUPID {
         case 0x0E:
             return "Advertising information. The specific usage is out of scope of this standard."
         case 0x0F:
-            return "Universal Resource Identifier (see [RFC 3986]). "
-        default:
+            return "Universal Resource Identifier (see [RFC 3986])."
+        case 0x10:
+            return "Universally unique identifier (see [RFC 4122])."
+        case 0x11...0xFF:
             return "Reserved for future standardization."
+        default:
+            return "UPID type is not currently defined."
         }
     }
 
     public var info: SegmentationUPIDInformation?
 
     init?(type: Int, length: Int, relevantBits: [Bit]) {
-        guard type <= 0xFF else { return nil }
```

Comment:
I chose to guard the ability to initialize the UPID only from the type integer values that are currently defined in the spec.  This value can be changed if any of the reserved types become defined in future specs.